src/sentry_plugins/vsts/client.py

```diff
@@ -20,7 +20,7 @@ def exception_means_unauthorized(self, exc):
     def request(self, method, path, data=None, params=None):
         headers = {
             'Accept': 'application/json; api-version={}'.format(self.api_version),
-            'Content-Type': 'application/json',
+            'Content-Type': 'application/json-patch+json',
```

Comment:
also, given this (I think) means 'application/json' and 'application/json-patch', our content type detection should probably use a library that does this for us, or should be improved